src/ComponentFolders/SignUpPage/SignUp.jsx

```diff
@@ -0,0 +1,146 @@
+          />
+          <div className="flex gap-x-7">
+            <input
+              type="password"
+              placeholder="   Password"
+              name="userPassword"
+              onChange={(e) => handleOnClick(e)}
+              className="h-14 w-56 broder-solid border-2 border-[#D1DBE3] rounded-md placeholder-gray-300"
+              value={signUpData.userPassword}
+            />
+            <input
+              type="password"
+              placeholder="   Confirm password"
+              name="userCondirmPassword"
+              onChange={(e) => handleOnClick(e)}
+              className="h-14 w-56 broder-solid border-2 border-[#D1DBE3] rounded-md placeholder-gray-300"
+              value={signUpData.userCondirmPassword}
+            />
+          </div>
+          <div className="flex items-center justify-between">
+            <p className="mr-7 ml-7 font-light text-[#9DAFBD]">Birth Date</p>
+            <input
+              type="text"
+              placeholder="  DD"
+              name="dayOfBirth"
+              onChange={(e) => handleOnClick(e)}
+              className="h-14 w-12 broder-solid border-2 border-[#D1DBE3] rounded-md placeholder-gray-300"
+              value={signUpData.dayOfBirth}
+            />
+            <input
+              type="text"
+              placeholder="  MM"
+              name="monthOfBirth"
+              onChange={(e) => handleOnClick(e)}
+              className="h-14 w-12 broder-solid border-2 border-[#D1DBE3] rounded-md placeholder-gray-300"
+              value={signUpData.monthOfBirth}
+            />
+            <input
+              type="text"
+              placeholder="  YYYY"
+              name="yearOfYear"
+              onChange={(e) => handleOnClick(e)}
+              className="h-14 w-12 broder-solid border-2 border-[#D1DBE3] rounded-md w-36 placeholder-gray-300"
+              value={signUpData.yearOfYear}
+            />
+          </div>
+          <div className="flex justify-around py-3 gap-8">
+            <button
+              type="submit"
+              className="bg-[#2DD3E3] font-medium text-2xl px-14 py-3 rounded-md shadow-[0px_7px_20px_rgba(0,0,0,0.2)]"
+            >
+              Login
+            </button>
+            <button
+              type="submit"
+              className="broder-solid border-2 border-[#2DD3E3] font-medium text-2xl px-14 rounded-md"
+            >
+              Signup
+            </button>
+          </div>
+        </form>
+        <div className="flex justify-around my-6">
+          <img src={lineImage} alt="A line" />
+          <p>Or</p>
+          <img src={lineImage} alt="A line" />
+        </div>
+        <div className="flex justify-center my-6 gap-x-20">
+          <img src={FacebookLogo} alt="Facebook logo" className='cursor-pointer'/>
+          <img src={GoogleLogo} alt="Google logo" className='cursor-pointer'/>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default SignUp;
```

Comment:
Test file is gone entirely :D can't see any test :D

Reply:
Not sure what is really goning on, all the tests are working fine at my side really, I will just commit and push it again